ops/lib.rs

```diff
@@ -618,14 +635,25 @@ fn is_result(ty: impl ToTokens) -> bool {
   }
 }
 
-fn is_string(ty: impl ToTokens) -> bool {
-  tokens(ty) == "String"
+fn is_string(ty: impl ToTokens) -> Option<bool> {
+  let toks = tokens(ty);
+  if toks == "String" {
+    return Some(false);
+  }
+  if toks == "& str" {
+    return Some(true);
+  }
+  None
 }
 
 fn is_option_string(ty: impl ToTokens) -> bool {
   tokens(ty) == "Option < String >"
 }
 
+fn is_cow_str(ty: impl ToTokens) -> bool {
+  tokens(&ty).starts_with("Cow <") && tokens(&ty).ends_with("str >")
+}
+
```

Comment:
`is_string` shouldn't also include Cow strings? Maybe it should be called `is_non_cow_string`?

It would be nice to have unit tests for these little utility functions. Is that hard?